rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -10696,23 +10728,27 @@ void opcode_0394(const ScriptArguments& args, const ScriptInt arg1) {
     @arg clearParticles Boolean true/false
 */
 void opcode_0395(const ScriptArguments& args, ScriptVec3 coord, const ScriptFloat radius, const ScriptBoolean clearParticles) {
-    GameWorld* gw = args.getWorld();
+    auto gw = args.getWorld();
```

Comment:
this should be a pointer

Reply:
Seems to me I ought to revert it back to `GameWorld* gw`. Do you agree?

Reply:
yes